common/aicore/path_finding.cpp

```diff
@@ -3439,28 +3380,6 @@ inline bool operator==(const pf_parameter &e1, const pf_parameter &e2)
   return true;
 }
 
-inline uint qHash(const pf_parameter &key, uint seed)
```

Comment:
Note to self: check that it's really not used

Reply:
It was added when the hash in `pf_reverse_map` was turned to `QHash`. This hasn't changed since. Maybe it's used indirectly in template instantiations?

Reply:
Looks like `-O2` also removes it, so it's really unused.